airflow/providers/opensearch/provider.yaml

```diff
@@ -0,0 +1,52 @@
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+---
+package-name: apache-airflow-providers-opensearch
+name: Opensearch
+description: |
+    `Opensearch <https://opensearch.org/>`__
+
+suspended: false
+versions:
+  - 1.0.0
+
+dependencies:
+  - apache-airflow>=2.4.3
```

Comment:
lets set 2.5.0
all provider releases after 19th of September 2023 will have 2.5.0
https://github.com/apache/airflow/blob/main/PROVIDERS.rst#id5

Reply:
If I do that then it fails the CI check for Airflow 2.4

Reply:
Hmm but it is still failing that check. Okay I’ll move that back to 2.5.0